.lintstagedrc.yml

```diff
@@ -0,0 +1,7 @@
+linters:
+  lib/*.{js, ts}:
+    - "npm run lint"
+    - "git add"
+  lib/*.{json,css,md}:
+    - "npm run format"
```

Comment:
hmm prettier is not running against your `json,css,md` in your `format` you ask it something different :confused:

Reply:
I fixed this in the latest :)